src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,109 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.23.0dev0 (11/03/2019)
+------------------------
```

Comment:
Needs one dash removed.

Reply:
Ah, sure. Doesn't look like that holds historically or break CI though :|

Happy to remove it, just trying to build up a sequence of checks that covers everything instead of needing you to check the markdown for me. You catch this in the rendered page on Github?

Reply:
CI should have failed w/o the edit. We used to run a doc publish check on one of the CI shards anyhow.